### ISSUE TITLE
blank screen after setup in fugu 14

### ISSUE THREAD

**Issue by luisferuc** [LinusHenze/Fugu14#140]
blank screen after setup in fugu 14

I tried to install fugu 14 with xcode from my mac, but when I open Fugu14 everything goes well, I give it in setup fugu14 and it installs it, I get the message "done" and then when I open fugu14 to give it unthetered, it only opens fugu with screen completely in white

today I tried with the method unc0ver + altstore and I had the same problem

someone has the same problem or was able to fix it?

Iphone 11 Pro Max A2218 - iOS 14.4.2
![IMG_3073](https://user-images.githubusercontent.com/93333725/139463494-43b27488-289a-43ed-bd6c-67a85cfc1b30.PNG)
![IMG_3075](https://user-images.githubusercontent.com/93333725/139463501-ba920099-23a0-4a33-ac79-b03c18b3611d.PNG)
![IMG_3074](https://user-images.githubusercontent.com/93333725/139463510-e18c5086-3568-43a4-a47d-8796a0bf8ec5.PNG)
![IMG_3072](https://user-images.githubusercontent.com/93333725/139461908-4a7739d9-3653-4b0f-a0ef-b0eedfbae261.PNG)

**Comment by VMMax** [comment]
> > > > > altstore will then prompt you to install unc0ver I believe, just follow the prompts, seeing the screenshot say "setup done, please open altstore to continue", why not just follow the instructions and see where it takes you?
> > > 
> > > 
> > > > > Any other issues please let me know.
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > Yea when you do it says unsupported
> > > 
> > > 
> > > did you happen to install the latest version of altstore / altserver on your mac/windows ? And then update the altsore ipa on your phone?
> > 
> > 
> > Yes
> 
> did you restore rootfs?

Can I when I have not jailbroken it?

**Comment by DogeLord081** [comment]
> the fugu14 app is only meant to be opened up once I believe, next part would be to open up altstore and follow the remaining steps.

No, after patching the fugu14 app, when you go back on altstore it says to open the fugu14 app again and follow the instructions there. That’s when the blank screen happens
